### DISCUSSION TITLE
Web admin: Tener la opción de agregar un logo a la companies

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#552] [3. Ideas & Feature Requests] [answered]
Hola a todos
Nos gustaria que el web admin se pueda colocar el logo de la company, para que, al usar el default.template.html de la rutina PBEmail, pueda ir el logo de la company

Actualmente aparece esto en el web admin:

![image](https://user-images.githubusercontent.com/61419471/143499089-6685e202-3f92-4e03-8703-e99ea75820c0.png)

plz @gmoplaza

Gracias =>

**Comment by acedward** [comment]
Si, es algo que tenemos que corregir. Abro bug issue.

**Comment by acedward** [comment, marked as answer]
https://github.com/Cotalker/documentation/issues/599
